packages/@react-types/statuslight/src/index.d.ts

```diff
@@ -10,10 +10,10 @@
  * governing permissions and limitations under the License.
  */
 
-import {DOMProps, StyleProps} from '@react-types/shared';
+import {AriaLabelingProps, DOMProps, StyleProps} from '@react-types/shared';
 import {ReactNode} from 'react';
 
-export interface SpectrumStatusLightProps extends DOMProps, StyleProps {
+export interface SpectrumStatusLightProps extends DOMProps, StyleProps, AriaLabelingProps {
```

Comment:
Does statuslight support aria labeling props? Seems a little strange to have a status light without a visible label

Reply:
If we don't want it, we should updated the [accessibility](https://react-spectrum.adobe.com/react-spectrum/StatusLight.html#accessibility) portion because we say to use `aria-label`.